specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -3640,6 +3669,16 @@
       "type": "string",
       "in": "path",
       "x-ms-skip-url-encoding": true
+    },
+    "RecognitionModel": {
+      "name": "RecognitionModel",
+      "description": "Recognition model name, maximum length is 128.",
+      "required": true,
```

Comment:
Please change this to be false.

And in future, we will change this as `enum`, let's do that then.

Reply:
Done.